addon/webextension/_locales/en_US/messages.json

```diff
@@ -0,0 +1,33 @@
+{
+  "pageShotDescription": { "message": "Page Shot takes clips and screenshots from pages, and can save a permanent copy of a page." },
+  "pageShotAuthors": { "message": "Ian Bicking, Donovan Preston, and Bram Pitoyo" },
+  "toolbarButtonLabel": { "message": "Take a shot" },
+  "selectorInstructions": { "message": "Drag or click on the page to select a region. Press ESC to cancel." },
+  "selectorMyShots": { "message": "My Shots" },
+  "selectorSave": { "message": "Save" },
+  "selectorSaveVisible": { "message": "Save visible" },
+  "selectorSaveFullPage": { "message": "Save full page" },
+  "selectorCancel": { "message": "Cancel" },
+  "selectorDownload": { "message": "Download" },
+  "contextMenuText": { "message": "Create Page Shot" },
+  "notificationLinkCopied": { "message": "Link Copied" },
```

Comment:
Can we use notificationLinkCopiedTitle to clarify the role of this string? That would spare us a localization comment in the .properties file